src/main/java/com/microsoft/spring/data/gremlin/conversion/GremlinScriptPropertiesLiteral.java

```diff
@@ -7,16 +7,15 @@
 
 import com.microsoft.spring.data.gremlin.common.Constants;
 import org.apache.commons.lang3.NotImplementedException;
-import org.springframework.util.Assert;
+import org.springframework.lang.NonNull;
```

Comment:
can lombok nonnull be replaced by this spring one?

Reply:
OK. And we will use spring framework in all code.